playbooks/mount_discovery_iso_for_pxe.yml

```diff
@@ -0,0 +1,6 @@
+- hosts: bastion
+  vars:
+    MOUNT_DISCOVERY_ISO: "{{setup_pxe_service | default(True)}}"
```

Comment:
This is making the assumption that setup_pxe is true if undefined. This should default to false.

Reply:
One more thing. Please put a space at the end of '{{' and the begining of '}}'